lib/actions/wizard.coffee

```diff
@@ -0,0 +1,118 @@
+_ = require('lodash-contrib')
+capitano = require('capitano')
+path = require('path')
+mkdirp = require('mkdirp')
+userHome = require('user-home')
+visuals = require('resin-cli-visuals')
+async = require('async')
+resin = require('resin-sdk')
+form = require('resin-cli-form')
+
+exports.wizard =
+	signature: 'quickstart [name]'
+	description: 'getting started with resin.io'
+	help: '''
+		Use this command to run a friendly wizard to get started with resin.io.
+
+		The wizard will guide you through:
+
+			- Create an application.
+			- Initialise an SDCard with the resin.io operating system.
+			- Associate an existing project directory with your resin.io application.
+			- Push your project to your devices.
+
+		Examples:
+
+			$ sudo resin quickstart
+			$ sudo resin quickstart MyApp
+	'''
+	root: true
+	permission: 'user'
+	action: (params, options, done) ->
+		async.waterfall [
+
+			(callback) ->
+				return callback() if params.name?
+
+				# TODO: Move this whole routine to Resin CLI Visuals
+				async.waterfall [
+
+					(callback) ->
+						resin.models.application.hasAny().nodeify(callback)
+
+					(hasAnyApplications, callback) ->
+						return callback(null, null) if not hasAnyApplications
+
+						async.waterfall [
+
+							(callback) ->
+								resin.models.application.getAll().nodeify(callback)
+
+							(applications, callback) ->
+								applications = _.pluck(applications, 'app_name')
+								applications.unshift
+									name: 'Create a new application'
+									value: null
+
+								form.ask
+									message: 'Select an application'
+									type: 'list'
+									choices: applications
+								.nodeify(callback)
+
+						], callback
+
+					(application, callback) ->
+						return callback(null, application) if application?
+
+						form.ask
+							message: 'Choose a Name for your new application'
+							type: 'input'
+						, (applicationName) ->
+							Promise.promisifyAll(capitano).run "app create #{applicationName}".return(applicationName)
+						.nodeify(callback)
+
+					(applicationName, callback) ->
+						params.name = applicationName
+						return callback()
+
+				], callback
+
+			(callback) ->
+				capitano.run("device init --application #{params.name}", callback)
+
+			(deviceName, callback) ->
+				params.device = deviceName
```

Comment:
@mkorakakis You can do the translation from `uuid` to `name` when necessary in the quick start command, since it's much easier to go from `uuid` to `name` than from `name` to `uuid`.